frontend/web/components/modals/CreateFlag.js

```diff
@@ -301,6 +301,7 @@ const CreateFlag = class extends Component {
     }
   }
   save = (func, isSaving) => {
+    console.log('calling save')
```

Comment:
log

Reply:
I forgot to push the commit removing that 😊